website/project/decorators.py

```diff
@@ -438,20 +439,6 @@ def wrapped(*args, **kwargs):
     # Return decorated function
     return wrapped
 
-def http_error_if_disk_saving_mode(func):
```

Comment:
To confirm, this function was removed because it isn't in use anywhere (based on a grep), is that correct?